README.md

```diff
@@ -1,2 +1,105 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [Video Walkthough](#video-walkthough)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [HTML-CSS-JS-capstone Project] <a name="about-project"></a>
+
+> The goal of this project is to create an online website for a conference with personalized content while demonstrating ability to create UIs adaptable to different screen sizes using media queries, using a CSS framework, applying best practices in HTML, CSS and JavaScript code.
+
+**HTML-CSS-JS-capstone Project** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This project use only html,css & javascript.
+<details>
+  <summary>Built with:</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>   
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> The key features of the website include:
+
+- **Responsive to Mobile and desktop**
+- **Use of semantic HTML tags**
+- **Uses objects to store and access data.**
+- **Applies JavaScript, CSS, and HTML best practices**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://c00p75.github.io/Capstone-project-Conference-page/home.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Video walkthough <a name="video-walkthough"></a>
+- [loom video link](https://www.loom.com/share/d3f223e30dbf48678c37bf38d9a03d31)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+👤 **Author1**
+
+- GitHub: [@coop75](https://github.com/c00p75)
+- LinkedIn: [George M'sapenda](http://www.linkedin.com/in/george-m-sapenda-593750201/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- The original design ideal by [Cindy Shin in Behance](https://www.behance.net/adagio07)
+
+- Project from [Microverse](https://www.microverse.org/?grsf=i6yi2m) html, css & Javascript module
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Great job @c00p75 🤝

- [ ] [Optional] Kindly fix your MIT license link to make your readme file more professional.